Client/Application/AppDelegate.swift

```diff
@@ -74,6 +74,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // Ecosia: lifecycle tracking
         Analytics.shared.activity(.launch)
         
+        // Ecosia: Engagement Service Initialization with AnalyticsId binding
+        ClientEngagementService.shared.initialize(parameters: ["id": User.shared.analyticsId.uuidString])
+        // Ecosia: Engagement Service consent request
+        ClientEngagementService.shared.requestAPNConsent(notificationCenterDelegate: self) { granted, error in
+            
```

Comment:
I guess this empty completion handler was just forgotten? Like I mentioned in the other PR, I find using async await more readable if there's no difference, also in that case we can use `try?` to make it clear we don't care about the error if that is the case, similar to the usage of `Statistics` just below.

Reply:
It was not forgotten actually 😅 .
As part of this ticket, there is no intention of performing any action upon denying/allowing the APNs.
There is a better explanation on the confluence page. The number of tickets worked at the moment in isolation does not include the Analytics work. In the `MOB-1863_braze_integration_all` there is a fully working implementation of the completion handler that includes the analytics sending.

Reply:
Got it, thanks for explaining! Did not realise the Analytics would happen here in the future, maybe we could add the TODO to make it clear or leave it without a completion handler for now.

Reply:
I believe I added it as I kept this approach for other stuff but then somehow removed it. Will re-add 😉